upstairs/src/lib.rs

```diff
@@ -2432,7 +2426,8 @@ impl BlockIO for Guest {
 
     /// Disable any more IO from this guest and deactivate the downstairs.
     async fn deactivate(&self) -> Result<(), CrucibleError> {
-        self.send_and_wait(BlockOp::Deactivate).await
+        let _ = self.send_and_wait(BlockOp::Deactivate).await?;
```

Comment:
Nit: should we assert that the result is `None` here?  It's paranoid, but could catch very weird mixups.